app/(main)/league/[leagueId]/entry/[entryId]/week/WeekHelper.tsx

```diff
@@ -61,6 +65,19 @@ export const onWeeklyPickChange = async ({
       userResults: updatedWeeklyPicks,
     });
 
+    const leagueEntryData = await getCurrentUserEntries(user.id, league);
+
+    const currentEntry = leagueEntryData.find(
+      (leagueEntry) => leagueEntry.$id === entry,
+    );
+
+    const currentEntrySelectedTeams = currentEntry?.selectedTeams || [];
+    currentEntrySelectedTeams[parseInt(week) - 1] = teamSelect;
```

Comment:
@shashilo making sure I understand this correctly, in an entry's selectedTeams array, does the index represent the team selection for the week? 

selectedTeams[0] equals the week 1 team selection for an entry

Reply:
That's correct. Because arrays start at 0 and weeks start at 1, they're off by 1 from one another.